script/Ciel/Day 5/05_01_CIEL05_4.txt

```diff
@@ -264,7 +264,7 @@ I'll happily get up close and personal with that bastard one more time if it mea
 [sha:b22b83954e63a1c8ef61012cf79ef404abc1c514]{
 -- Page 692, Offset 12700.
 -- 「とにかく任せて！　一緒にやらせてくれ、先輩！」
-"Look, just leave it to me!
+"In the meantime, just leave it to me!
```

Comment:
Gotta say I _really_ don't like literal translations of tonikaku. People don't say this!